CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 - Refactoring stories to simpler setup.
 - Linting fixes.
 
+- Added footer component
+- Added copyright component
+
```

Comment:
I'm not sure if it would make difference for this kind of a small change log, but maybe it would be nice to sort in in alphabetical order.

Reply:
It should be added but according to keep a changelog way, with titles etc.

Reply:
We can do that when releasing a new version? Not sure we would benefit from this while the list of changes is in `Unreleased`.